modules/zpca/zpca.py

```diff
@@ -0,0 +1,149 @@
+###############################################################################
+#
+#   Main script which does the job of parsing values and plotting graphs
+#   for zpca values
+#
+#   AUTHOR: Krish Agarwal
+#   AFFILIATION: University_of_Basel
+#   CONTACT: akrish136@gmail.com
+#   CREATED: 15-03-2021
+#   LICENSE: Apache_2.0
+#
+###############################################################################
+
+from __future__ import print_function
+import logging
+
+from multiqc.plots import scatter
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+    This class is instantiated in setup.py file and contains
+    all the functions required by the zpca plugin.
+    """
+
+    def __init__(self):
+        """
+        Initializes the parent function of MultiQC and calls
+        relevant functions for the searching of files and the
+        plotting of graphs.
+        """
+
+        super(MultiqcModule, self).__init__(
+            name="ZPCA",
+            anchor="zpca",
+            href="TO ADD",  # TO DO: Add link and description
```

Comment:
todo?

Reply:
Ah right, I just saw your question regarding this field.
Well, the link is the GH repo: https://github.com/zavolanlab/zpca
For the description we have to ask @fgypas , as I cannot find anything in his README file.